spec/services/proofing/resolution/progressive_proofer_spec.rb

```diff
@@ -8,7 +8,17 @@
   let(:threatmetrix_session_id) { SecureRandom.uuid }
   let(:timer) { JobHelpers::Timer.new }
   let(:user) { create(:user, :fully_registered) }
+
   let(:instant_verify_proofer) { instance_double(Proofing::LexisNexis::InstantVerify::Proofer) }
+
+  let(:instant_verify_proofer_result) { nil }
+
+  let(:aamva_proofer) do
+    double = instance_double(Proofing::Aamva::Proofer)
+    allow(double).to receive(:proof)
+    double
```

Comment:
```suggestion
    instance_double(Proofing::Aamva::Proofer, proof: nil)
```

Reply:
I'll make this and the other spec changes you suggest, but at @matthinz's request, they'll be in a seperate PR. This one will be just the missing ID handling and the changes to fix that. I'll invoke you as a reviewer on the cleanup PR.